src/emulator/human.py

```
@@ -386,6 +404,31 @@ def click_element(self, element, scroll_into_view=True):
+                pass
+            self.move_to_element(element, scroll_into_view=False)
+            self._draw_debug_cursor(
+                self.last_mouse_position[0], self.last_mouse_position[1], color="green"
+            )
+            try:
+                self.driver.execute_script("arguments[0].click();", element)
+            except Exception:
+                try:
+                    element.click()
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)